swarm/network/simulation/kademlia_test.go

```diff
@@ -63,4 +72,87 @@ func TestWaitTillHealthy(t *testing.T) {
 			t.Fatal(err)
 		}
 	}
+
+	// now create a snapshot of this network
+	snap, err := sim.Net.Snapshot()
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	// create a control simulation
+	controlSim := New(createSimServiceMap(false))
+	defer controlSim.Close()
+
+	// load the snapshot into this control simulation
+	err = controlSim.Net.Load(snap)
+	if err != nil {
+		t.Fatal(err)
+	}
+	_, err = controlSim.WaitTillHealthy(ctx)
+	if err != nil {
+		t.Fatal(err)
+	}
+	// for each node...
+	nodeIDs := controlSim.UpNodeIDs()
+	if len(nodeIDs) != testNodesNum {
```

Comment:
this check is not needed since nodeIds is set with `AddXXX(testNodesSum)` which is unit tested to be correct